packages/ra-ui-materialui/src/list/ListView.tsx

```diff
@@ -191,7 +191,7 @@ export interface ListViewProps {
      *     </List>
      * );
      */
-    children: ReactNode;
+    children?: ReactNode;
```

Comment:
Why ?

Reply:
To fix a ts error, but now that you mention it. It was not the right fix